internal/internal_worker_base.go

```diff
@@ -538,6 +541,14 @@ func (bw *baseWorker) logPollTaskError(err error) {
 		bw.lastPollTaskErrStarted = time.Now()
 		return
 	}
+
+	// Ignore connection loss on server shutdown. This helps with quiescing spurious error messages
+	// upon server shutdown (where server is using the SDK).
+	st, ok := status.FromError(err)
+	if ok && st.Code() == codes.Unavailable && strings.Contains(st.Message(), "graceful_stop") {
+		return
+	}
```

Comment:
This is unrelated, but, just wanted to do it while I was in here - helps shut up some pointless logspew in the dev server.

Reply:
but a normal user would want to get logs if the poller received errors from the server is shutting down

Reply:
Removing this for now and will add in another PR once I learn from server how I can detect we're an internal worker.